### DISCUSSION TITLE
Req. Assistance in Installation

### DISCUSSION THREAD

**Discussion by Ram-Gangadhar** [kube-green/kube-green#391] [Q&A]
I have followed the steps which are mentioned in website [ https://kube-green.dev/ ] , but I'm encountering few errors like this
Error from server (BadRequest): error when creating "working-hours.yml": SleepInfo in version "v1alpha1" cannot be handled as a SleepInfo: strict decoding error: unknown field "spec.patchesJson6902" 

Can you please let me know, if I missed anything?

**Comment by davidebianchi** [comment]
It seems you are installing a SleepInfo improvement in main, but not yet released. Can you try to remove the patchesJson6902 field?
Or if you share the CRD, I can help you to correct it

**Reply by Ram-Gangadhar** [reply]
I download repo from https://kube-green.dev/docs/getting-started/ link . as per your recommendation above removed  patchesJson6902 field this one and tried Running the tool  how ever still getting error. please let me your availability so that resolve this issue over the meeting.
